docs/v0.6.0/riofile.md

```diff
@@ -526,3 +523,62 @@ guestbook   https://guestbook-default.enu90s.on-rio.io
 We can now access this endpoint over encrypted https!
 
 
+#### Using answer file 
```

Comment:
Good stuff 👍 . There is a section above this that shows small template usage but this is better.